modules/ROOT/pages/authentication-authorization/built-in-roles.adoc

```diff
@@ -45,11 +45,11 @@ In other words, the `editor` role cannot add to the schema but can only make cha
 <<access-control-built-in-roles-admin, `admin`>>::
 * Access to all databases.
 * Traverse, read, and write on the data graph.
-* Allows loading data.
+* Load data.
```

Comment:
should you do these updates for `PUBLIC` as well and not just `admin`?

Reply:
Of course, thanks!